src/python/pants/backend/python/dependency_inference/default_module_mapping.py

```diff
@@ -122,14 +124,9 @@ def two_groups_hyphens_two_replacements_with_suffix(
     "delta-spark": ("delta",),
     "django-activity-stream": ("actstream",),
     "django-cors-headers": ("corsheaders",),
-    "django-countries": ("django_countries",),
-    "django-filter": ("django_filters",),
-    "django-fsm": ("django_fsm",),
     "django-notifications-hq": ("notifications",),
     "django-oauth-toolkit": ("oauth2_provider",),
-    "django-object-actions": ("django_object_actions",),
     "django-postgres-extra": ("psqlextra",),
-    "django-redis": ("django_redis",),
```

Comment:
Ah, sorry, these ones have to remain, because there's a `django` regex pattern in `DEFAULT_MODULE_PATTERN_MAPPING` that overrules the catch-all `-` -> `_` rule. That pattern converts `django-abc-def` -> `abc_def`, stripping off the `django-`.